Auth0/JWTAlgorithm.swift

```diff
@@ -49,7 +47,6 @@ enum JWTAlgorithm: String {
             guard let publicKey = jwk.rsaPublicKey, let rsa = A0RSA(key: publicKey) else { return false }
             let sha256 = A0SHA()
             return rsa.verify(sha256.hash(data), signature: signature)
```

Comment:
Does it still make sense for this to be in a `switch` given it only has one option, or can it be simplified?